charts/router/templates/router-deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.global.experimental_native_ingress }}
```

Comment:
I added the `global.experimental_native_engress` value to the `values.yaml` for the router chart.. this should support a standalone installation of the router